packages/core/strapi/src/core/app-configuration/config-loader.ts

```diff
@@ -9,7 +9,17 @@ export default (dir: string) => {
 
   return fs
     .readdirSync(dir, { withFileTypes: true })
-    .filter((file) => file.isFile() && VALID_EXTENSIONS.includes(path.extname(file.name)))
+    .filter((file) =>{
+      const fullPath = path.join(dir, file.name);
+      let stat: fs.Stats | fs.Dirent;
+      try {
+        stat = file.isSymbolicLink() ? fs.statSync(fullPath) : file;
+      } catch(err) {
```

Comment:
```suggestion
      } catch (err) {
```
linting

Reply:
I've fixed linting issues, `npm run lint:other` on both files pass for me